react/features/notifications/middleware.js

```diff
@@ -29,15 +31,27 @@ MiddlewareRegistry.register(store => next => action => {
     switch (action.type) {
     case PARTICIPANT_JOINED: {
         const result = next(action);
-
         const { participant: p } = action;
+        const { dispatch, getState } = store;
 
         if (!p.local && !joinLeaveNotificationsDisabled()) {
-            store.dispatch(showParticipantJoinedNotification(
-                getParticipantDisplayName(store.getState, p.id)
+            dispatch(showParticipantJoinedNotification(
+                getParticipantDisplayName(getState, p.id)
             ));
         }
 
+        if (!interfaceConfig.DISABLE_FOCUS_INDICATOR && p.role === PARTICIPANT_ROLE.MODERATOR) {
```

Comment:
this will break mobile I think. `interfaceConfig` is undefined

Reply:
Good point! It's fixed now!